human_analysis/nbconverted/2_identify_generic_genes_pathways.py

```diff
@@ -282,6 +293,7 @@
                                              shared_gene_rank_scaled_df,
                                              1000)
     print(r, p, ci_high, ci_low)
+    # 0.2397233887765045 3.960267987659121e-223 0.22519096058775306 0.25479874632262184
```

Comment:
This seems like something you might want to put in your notebook in `tests/2_identify_generic_genes_pathways.ipynb`, doesn't it? You could have an assert statement making sure the correlation is what you expect it to be.

Reply:
Yes, good point. I have added an assertion here